model.go

```diff
@@ -49,3 +52,11 @@ func (p PullRequestEvent) Issue() Issue {
 func (i Issue) FullName() string {
 	return fmt.Sprintf("%s/%s#%d", i.Repository.Owner, i.Repository.Name, i.Number)
 }
+
+func internalRepositoryRepresentation(githubRepository *github.Repository) Repository {
```

Comment:
Couldn't we name the function simply `repository` instead?
External representations should be namespaced (like `github.Repository`). It should be implicit that all other representations are internal and thus use of the `internal<X>Representation` redundant.